src/components/StorageOverview/Inventory/utils.js

```diff
@@ -0,0 +1,74 @@
+import { getVmStatus, VM_STATUS_ALL_ERROR, VM_STATUS_ALL_PROGRESS } from '../../../utils/status/vm';
```

Comment:
this file is the same as `ClusterOverview/Inventory/utils` so you could move it to `Dashboard/Inventory/utils` and reuse it in `ClusterOverview/Inventory` and `StorageOverview/Inventory`

Reply:
maybe `mapStatuses` and `resolveStatusResult` could be moved. But I am not sure about the other mapXToProps. Shouldn't these be different for each dashboard?

Reply:
as written in https://github.com/kubevirt/web-ui-components/pull/311#discussion_r272602384 I think that its fine to have overlapping information so IMHO it makes sense to move these utils as other dashboards will use it.

Reply:
if that is the case then sure, it makes sense to move them